### PR TITLE
[controller] Add leaderFollowerModelEnabled to StoreInfo for Controller::ControllerClient compatibility

### PR DESCRIPTION
**Pull request by sushantmane**
##  Add leaderFollowerModelEnabled to StoreInfo for Controller::ControllerClient compatibility

Resolves #XXX

## Does this PR introduce any user-facing changes?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure to explain your proposed changes and call out the behavior change.

### CODE REVIEW

File: internal/venice-common/src/main/java/com/linkedin/venice/meta/StoreInfo.java [hunk 1]
```diff
@@ -726,4 +729,13 @@ public boolean isActiveActiveReplicationEnabled() {
   public void setActiveActiveReplicationEnabled(boolean activeActiveReplicationEnabled) {
     this.activeActiveReplicationEnabled = activeActiveReplicationEnabled;
   }
+
+  public boolean isLeaderFollowerModelEnabled() {
+    return leaderFollowerModelEnabled;
```
Comment (huangminchn) [inline internal/venice-common/src/main/java/com/linkedin/venice/meta/StoreInfo.java:734]: How about just always return true?

**Review by huangminchn — CHANGES_REQUESTED** [review]

File: internal/venice-common/src/main/java/com/linkedin/venice/meta/StoreInfo.java (hunk 1 above)
Comment (sushantmane) [reply internal/venice-common/src/main/java/com/linkedin/venice/meta/StoreInfo.java:734]: that's better

**Review by huangminchn — APPROVED** [review]
Thanks for the quick turnaround Sushant!

**Comment by sushantmane** [comment]
Thanks for the review, Min!

**Pull request merged**